elements/pfe-textinput/src/pfe-textinput.html

```diff
@@ -0,0 +1,2 @@
+<slot></slot>
+<span></span>
```

Comment:
It might be nice to add a class or ID on this span to identify what it's for

Reply:
This isn't needed anymore since we're just going to use a slot for the HTML.